config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   get "/hello_react", to: "hello_react#contracts"
+
+  resources :applicants
+  resources :offers
+  resources :contracts
+  get "offers/instructor/:instructor_id" => "offers#show_by_instructor"
+  post "offers/:id/send-contract" => "offers#send_contract"
+  post "contracts/:id/nag" => "contracts#nag"
```

Comment:
You can [nest these](http://guides.rubyonrails.org/routing.html#adding-more-restful-actions) under `resources`